### PR TITLE
Improve wording and typespecs in ambiguous datetimes

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: lib/elixir/lib/calendar/datetime.ex [hunk 1]
```diff
@@ -109,8 +109,9 @@ defmodule DateTime do
       {:ok, ~U[2016-05-24 13:26:08.003Z]}
 
   When the datetime is ambiguous - for instance during changing from summer
-  to winter time - the two possible valid datetimes are returned. First the one
-  that happens first, then the one that happens after.
+  to winter time - the two possible valid datetimes are returned in a tuple,
+  being its first element the datetime that takes place first, and its second element
```
Comment (josevalim) [inline lib/elixir/lib/calendar/datetime.ex:113]: This is a long sentence. We should probably keep the period.

Comment (eksperimental) [reply lib/elixir/lib/calendar/datetime.ex]: Fixed.

File: lib/elixir/lib/calendar/datetime.ex [hunk 2]
```diff
@@ -364,8 +365,9 @@ defmodule DateTime do
       {:ok, ~U[2016-05-24 13:26:08.003Z]}
 
   When the datetime is ambiguous - for instance during changing from summer
-  to winter time - the two possible valid datetimes are returned. First the one
-  that happens first, then the one that happens after.
+  to winter time - the two possible valid datetimes are returned in a tuple.
+  Its first element is the datetime that takes place first, and its second element
+  the one that takes place after.
```
Comment (josevalim) [inline lib/elixir/lib/calendar/datetime.ex:370]: This is ambiguous. The first element of the tuple is an atom! :)

```suggestion
  The first datetime is also the one who comes first chronologically, while the
  second element comes last.
```

Comment (eksperimental) [reply lib/elixir/lib/calendar/datetime.ex:370]: True, but still we cannot say "second element", we need to say "while the second datetime comes last"

Comment (eksperimental) [reply lib/elixir/lib/calendar/datetime.ex:370]: or "the second one comes last"

Comment (josevalim) [reply lib/elixir/lib/calendar/datetime.ex:370]: Yup, updated.

File: lib/elixir/lib/calendar/datetime.ex [hunk 3]
```diff
@@ -109,8 +109,9 @@ defmodule DateTime do
...
```
Comment (eksperimental) [inline lib/elixir/lib/calendar/datetime.ex:114]:
```suggestion
  The first datetime is also the one who comes first chronologically, while the
  second one comes last.
```

File: lib/elixir/lib/calendar/datetime.ex [hunk 4]
```diff
@@ -109,8 +109,9 @@ defmodule DateTime do
       {:ok, ~U[2016-05-24 13:26:08.003Z]}
 
   When the datetime is ambiguous - for instance during changing from summer
-  to winter time - the two possible valid datetimes are returned. First the one
-  that happens first, then the one that happens after.
+  to winter time - the two possible valid datetimes are returned in a tuple.
+  The first datetime is also the one who comes first chronologically, while the
+  second one comes last.
```
Comment (josevalim) [inline lib/elixir/lib/calendar/datetime.ex:114]:
```suggestion
  The first datetime is also the one which comes first chronologically, while
  the second one comes last.
```

File: lib/elixir/lib/calendar/datetime.ex [hunk 5]
```diff
@@ -364,8 +365,9 @@ defmodule DateTime do
...
```
Comment (josevalim) [inline lib/elixir/lib/calendar/datetime.ex:370]:
```suggestion
  The first datetime is also the one which comes first chronologically, while
  the second one comes last.
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**